dbt/models/iasworld_test/schema.yml

```diff
@@ -0,0 +1,122 @@
+sources:
```

Comment:
We don't have access to the full set of tables that we do from the prod environment. I also don't know to what extend the test table schemas differ from prod. A brief look shows that they mostly seem identical, but I know they sometimes add new columns in test first.